pytradfri/command.py

```diff
@@ -14,12 +14,12 @@ def __init__(
         self,
         method: str,
         path: list[str],
-        data: dict[str, Any] | None = None,
+        data: Any | None = None,
         *,
         parse_json: bool = True,
         observe: bool = False,
         observe_duration: int = 0,
-        process_result: TypeProcessResultCb = None,
+        process_result: Optional[Any] = None,
```

Comment:
We should update `TypeProcessResultCb`. We still know it's a callback. It's just the return value we want to be more lax about.

Reply:
> I think the library should represent the tradfri api as close as possible. I don't think we should remove it.

OK, then it'll remain.